api/src/common/dto/get-applications.dto.ts

```diff
@@ -29,5 +27,9 @@ export class GetApplicationsDto extends PaginationDto {
   @ApiPropertyOptional({ enum: ApplicationSortOptions })
   @IsOptional()
   @IsEnum(ApplicationSortOptions)
-  orderBy = ApplicationSortOptions.FACILITY_NAME;
+  orderBy = ApplicationSortOptions.SUBMISSION_ID;
+
+  @IsOptional()
+  @IsString()
```

Comment:
should this be `@IsNumber()`?

Reply:
It comes back as string, even in application entity it's `string` with type _money_